src/apic-extension/azext_apic_extension/tests/latest/test_api_commands.py

```diff
@@ -184,6 +184,18 @@ def test_examples_update_api(self):
             self.check('summary', 'Basic REST API service'),
         ])
 
+    @ResourceGroupPreparer(name_prefix="clirg", location=TEST_REGION, random_name_length=32)
+    @ApicServicePreparer()
+    @ApicApiPreparer()
+    @ApicMetadataPreparer()
+    def test_examples_update_single_custom_metadata(self, metadata_name):
+        self.kwargs.update({
+            'customProperty': '{}'.format(metadata_name),
+        })
+        self.cmd('az apic api update -g {rg} -n {s} --api-id {api} --set customProperties.{customProperty}=false', checks=[
```

Comment:
Minor: The `ApicMetadataPreparer` sets the metadata name to kwargs `m` (see https://github.com/blackchoey/azure-cli-extensions/blob/main/src/apic-extension/azext_apic_extension/tests/latest/utils.py#L53).

You can change the code to `self.cmd('az apic api update -g {rg} -n {s} --api-id {api} --set customProperties.{m}=false'` and remove the `self.kwargs.update(...)` code to simplify the test case.

Reply:
very good suggestion. optimized